testing/matrix/linter/rules/rules.go

```diff
@@ -85,6 +90,7 @@ type ObjectLinter struct {
 	Module         utils.Module
 	Values         string
 	EnabledModules map[string]struct{}
+	HelmTemplates  []*chart.File
```

Comment:
This property is not used anywhere :thinking:

Reply:
Thanks, this one from first version